docs/routing/shallow-routing.md

```diff
@@ -54,7 +54,7 @@ componentDidUpdate(prevProps) {
 
 ## Caveats
 
-Shallow routing **only** works for same page URL changes. For example, let's assume we have another page called `pages/about.js`, and you run this:
+Shallow routing **only** works for URL changes in the same, current page. For example, let's assume we have another page called `pages/about.js`, and you run this:
```

Comment:
Could you expand on what this means?

Reply:
@leerob shallow routing works only for the page you're currently in, and as long as it's that same page where you'll be navigating to, this also complements better with the paragraph below where it does mention the "current" page. (This PR was created based on customer feedback, but if it's already clear enough then it's fine)